.idea/src/lesson11/Task1/Array.java

```diff
@@ -0,0 +1,24 @@
+package lesson11.Task1;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class Array {
+    public static void main(String[] args) {
+        List<Integer> list = new ArrayList<Integer>();
```

Comment:
List<Integer> list = new ArrayList<**Integer**>(); can be simplified : 
List<Integer> list = new ArrayList<>();